src/helpers.js

```diff
@@ -57,7 +57,12 @@ exports.getTopic = (attributes, slots) => {
     }
 };
 
-exports.getPodcastDirective = (podcastUrl) => {
+exports.getStage = (functionName) => {
```

Comment:
This file is beginning to look like a bit of mashup of everything. Maybe we can start grouping some of the functions out into logical files.